### ISSUE TITLE
ARCore support in moto G6 But Need to restart phone every time

### ISSUE THREAD

**Issue by jayesh8585** [google-ar/arcore-unity-sdk#376]
I have a moto G6, I'm using unity 2018.2 and latest version of ARCore 1.4 but when I create an android build it not support.

Note: I have read for android SDK that we have add armeabi-v7 in gradle setting, so is ther anything like this we have do with unity?

Thanks :)

**Comment by pablisho** [comment]
Hi, thanks for the report. I reported it internally and I'll be back with a response.

**Comment by jayesh8585** [comment]
I have checked again It is working but every time need to restart my phone.
I think it is major issue for restarting phone.
It support moto G6 but need to restart every time.

**Comment by tinatotty91** [comment]
The behaviour is still happening even with version 1.5 of ARCore. I'm using a Moto G6 for testing, Unity version 2018.3.0b2 and version 1.5 of the arcore sdk for unity and the android sdk. When I build the app on the phone, every feature of ARCore (motion tracking, plane detection etc) works only after restarting it, while it works perfectly using Unity Remote.

**Comment by pablisho** [comment]
Thanks for confirming that restarting the phone fixes the problem. I'm marking this one as a duplicate of #373.
